source/guides/guides/cross-browser-testing.md

```diff
@@ -9,7 +9,11 @@ Tests that require the {% url "`chromeWebSecurity` configuration option to be di
 {% endnote %}
 
 {% note warning 'Firefox Garbage Collection' %}
-Cypress triggers Firefox's internal garbage collection (GC) to better manage the browser's memory consumption. Since this process has some overhead it is disabled in **open mode** and enabled by default in **run mode** to offer the best experience locally and in CI. The Firefox garbage collection behavior is configurable via the {% url "`firefoxGcInterval` configuration" configuration#Global %} value, and you can learn more {% url "here" configuration#firefoxGcInterval %}.
+Cypress triggers Firefox's internal garbage collection (GC) to better manage the browser's memory consumption. {% url "Learn more here" configuration#firefoxGcInterval %}.
+{% endnote %}
+
+{% note warning 'Firefox Headless Video Recording' %}
+Video recording is not currently supported in the **Firefox** headless browser. See {% issue 1767 %} for more details.
```

Comment:
I don't think it's best to lead or even include this. Since many users will confuse "headless" with "run mode." Looking at https://github.com/cypress-io/cypress/issues/1767, we don't support FF headless anyways.

Reply:
We also mentioned this in the command line doc, which is sufficient: https://github.com/cypress-io/cypress-documentation/blob/v4.0/source/guides/guides/command-line.md#L212-L216

Reply:
Ok, yeah, I am used to headless being the default in run mode and actually this point needs to be mentioned in the command line doc too.